help/de/Help_ch_Account-Actions.xml

```diff
@@ -253,173 +375,192 @@
-        <para>Bargeld: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten sein.
-        </para>
-      </listitem>
-
-      <listitem>
-        <para>Aktiva: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten sein.
-        </para>
-      </listitem>
-
-      <listitem>
-        <para>Kreditkarte: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten sein.
-        </para>
-      </listitem>
 
-      <listitem>
-        <para>Fremdkapital: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten sein.
-        </para>
-      </listitem>
-
-      <listitem>
-        <para>Aktienkonto: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten sein.
-        </para>
-      </listitem>
-
-      <listitem>
-        <para>Investmentfonds: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten
-          sein.
-        </para>
-      </listitem>
-
-      <listitem>
-        <para>Ertrag: Unterkonto können nur Aufwands- und Ertragskonten sein.
-        </para>
-      </listitem>
+            <entry namest="c3">
+              <para>Devisenhandelskonten repräsentieren/gleichen aus das schwebende Ergebnis Ihres Wertpapier- und
+                Devisenhandels. Sobald sie den ersten Kauf tätigen, etwa Umtausch Ihrer
+                Heimwährung in die des Urlaubslandes, wird für beide Währungen ein Konto
+                angelegt, sofern es nicht bereits existiert. Die Details sind in
+                <ulink url="&tit-sel-tut;"></ulink> und <ulink url="&tit-sel-gnc;"></ulink> in
+                Englisch beschrieben.
+              </para>
+            </entry>
 
-      <listitem>
-        <para>Aufwand: Unterkonto können nur Aufwands- und Ertragskonten sein.
-        </para>
-      </listitem>
+            <entry namest="c4">
+              <para>nur Devisenhandelskonten
+              </para>
+            </entry>
+          </row>
 
-      <listitem>
-        <para>Eigenkapital: Unterkonto können nur Eigenkapitalkonten sein.
-        </para>
-      </listitem>
+          <row rowsep="1">
+            <entry namest="c1">
+              <para>Fehler
+              </para>
+            </entry>
 
-      <listitem>
-        <para>Offene Forderungen: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten
-          sein.
-        </para>
-      </listitem>
+            <entry namest="c2">
+              <para>Ausbuchungskonto-<replaceable>XXX</replaceable>, Ausgleichskonto-<replaceable>XXX</replaceable>
+              </para>
+            </entry>
 
-      <listitem>
-        <para>Offene Verbindlichkeiten: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und
-          Ertragskonten sein.
-        </para>
-      </listitem>
+            <entry namest="c3">
+              <para>Sie haben mindestens eine unausgeglichene Buchung eingegeben. [wann welcher Name?] Zur Behebung
```

Comment:
`[wann welcher Name?]` Kannst du das noch klären?

Reply:
nee, hier fehlt ein Komma. gemeint ist "... steuerabzugsfähig oder nicht steuerabzugsfähig ..."

Reply:
Meinst du, in welcher Situation wird in welches Konto (Ausbuchungskonto bzw. Ausgleichskonto) gebucht? 

Geht das an dieser Stelle nicht etwas zu weit und müsste eher in den Abschnitt "Arbeiten mit Buchungen" (Transactions.xml)?

Reply:
dann den Klammerausdruck weg